app/models/navigation/Menus.scala

```diff
@@ -22,7 +22,11 @@ object Menus {
       "Create".clusterRouteMenuItem(cluster)),
       None),
     "Preferred Replica Election".clusterMenu(cluster),
-    "Reassign Partitions".clusterMenu(cluster)
+    "Reassign Partitions".clusterMenu(cluster),
+    Menu("Logkafka",IndexedSeq(
```

Comment:
We should add the ability to enable/disable menu items based on cluster config.  If you'd like to attempt to make these changes, please do, otherwise I'll get around to it.